op-geth-proxy/geth-proxy.go

```diff
@@ -36,11 +37,20 @@ type rpcMessage struct {
 }
 
 func ForwardToSequencer(message rpcMessage) {
+	var hexString string
+	if err := json.Unmarshal(message.Params[0], &hexString); err != nil {
+		panic(err)
```

Comment:
We should not panic based on user input. Better to log the error and do nothing, the client can then retry and the server will still be alive

Reply:
Seems like there is some panic recovery built in to this http request handler (sending invalid bytes doesn't kill the server). Still, I think the stack trace is unnecessary here, so I'll switch to logging